### PR TITLE
Retrieve samples parameter on 'comp.task' RPC

### PR DESCRIPTION
**Pull request by mdtanrikulu**

### CODE REVIEW

**Review by shadeofblue — APPROVED** [review]
wolfofwallstreet

**Comment by shadeofblue** [comment]
though @mdtanrikulu please add or modify a test to make sure that this value is covered

File: tests/apps/blender/task/test_blenderrendertask.py [hunk 1]
```diff
@@ -650,7 +650,8 @@ def _task_dictionary(self):
                 "resolution": [
                     320,
                     240
-                ]
+                ],
+                "samples": 0
```
Comment (shadeofblue) [inline tests/apps/blender/task/test_blenderrendertask.py]: could I suggest using a value different than 0? @mdtanrikulu

Comment (mdtanrikulu) [reply tests/apps/blender/task/test_blenderrendertask.py]: normally there was no samples parameter at all, but sure I updated it.

**Pull request merged**